tensorflow/BUILD

```diff
@@ -272,6 +272,12 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
```

Comment:
i think this is a leftover?

Reply:
Yes, didn't notice that was still there. Thanks!